### ISSUE TITLE
Is STL read/write support intended to be supported for long term

### ISSUE THREAD

**Issue by shahchintan14** [3MFConsortium/lib3mf#153]

**Comment by shahchintan14** [comment]
Could we use lib3mf for reading and writing STL files in production? Is STL support going to be enhanced/maintained if required in future?

**Comment by alexanderoster** [comment]
Yes, it will be maintained and enhanced. An obvious candidate for enhancements of course would be support for ASCII STLs. We also have OBJ Support somewhere on the roadmap.

**Comment by shahchintan14** [comment]
@alexanderoster This is great! Is there a timeline for ASCII STLs and OBJ? Would the plan for OBJ be support for read and write both?

**Comment by martinweismann** [comment]
There is no explicit timeline for ASCII STL or OBJ-read/write support.
